src/EVM/UnitTest.hs

```diff
@@ -47,6 +48,7 @@ import System.IO (hFlush, stdout)
 import Witch (unsafeInto, into)
 import Data.Vector qualified as V
 import Data.Char (ord)
+import Control.Monad.State.Strict (runStateT)
```

Comment:
Can you move merge this import with the other `Control.Monad.State.Strict` 24 lines above? If it is really needed...